tests/admin_widgets/tests.py

```diff
@@ -1093,30 +1097,30 @@ def execute_basic_operations(self, mode, field_name):
         ])
 
         # Choose some more options --------------------------------------------
-        self.get_select_option(from_box, str(self.peter.id)).click()
-        self.get_select_option(from_box, str(self.lisa.id)).click()
+        self.select_option(from_box, str(self.peter.id))
+        self.select_option(from_box, str(self.lisa.id))
 
         # Confirm they're selected after clicking inactive buttons: ticket #26575
         self.assertSelectedOptions(from_box, [str(self.peter.id), str(self.lisa.id)])
         self.selenium.find_element_by_id(remove_link).click()
         self.assertSelectedOptions(from_box, [str(self.peter.id), str(self.lisa.id)])
 
         # Unselect the options ------------------------------------------------
-        self.get_select_option(from_box, str(self.peter.id)).click()
-        self.get_select_option(from_box, str(self.lisa.id)).click()
+        self.select_option(from_box, str(self.peter.id))
+        self.select_option(from_box, str(self.lisa.id))
 
         # Choose some more options --------------------------------------------
-        self.get_select_option(to_box, str(self.jason.id)).click()
-        self.get_select_option(to_box, str(self.john.id)).click()
+        self.select_option(to_box, str(self.jason.id))
+        self.select_option(to_box, str(self.john.id))
 
         # Confirm they're selected after clicking inactive buttons: ticket #26575
         self.assertSelectedOptions(to_box, [str(self.jason.id), str(self.john.id)])
         self.selenium.find_element_by_id(choose_link).click()
         self.assertSelectedOptions(to_box, [str(self.jason.id), str(self.john.id)])
```

Comment:
The changes here seem to break the assertion on this line. (Ref e00d77c4834b40f06f9bf271da5fdfb526ad8f56)

Reply:
Ah it took me a while. Of course, it only selects. Some clicks have been deselects tho. I added to explicit methods.